filament/backend/include/private/backend/HandleAllocator.h

```diff
@@ -363,6 +393,7 @@ class HandleAllocator {
     // Below is only used when running out of space in the HandleArena
     mutable utils::Mutex mLock;
     tsl::robin_map<HandleBase::HandleId, void*> mOverflowMap;
+    tsl::robin_map<HandleBase::HandleId, std::string> mDebugTags;
```

Comment:
yeah I Would use CString for this, it has less overhead because it's not trying to do the small string optimization, which we don't need here.